docs/contributing.md

```diff
@@ -50,13 +52,15 @@ for this purpose.
 
 ### Testing Changes Locally
 
-To run a dag file in a temporary local environment, use the `local-airflow.sh`:
+To run a dag file in a temporary local environment, use `local-airflow.sh`. The script will symlink just the DAG provided to speed up parsing times.
```

Comment:
maybe also mention it'd better to develop on a cloudtop machine?

Reply:
I'll add some instructions to set up VSCode with SSH to cloudtop in our internal onboarding doc.